tests/unit/test_config.js

```diff
@@ -27,4 +28,18 @@ describe(__filename, () => {
     conf = require('config');
     expect(parseInt(conf.get('serverPort'), 10)).toEqual(5000);
   });
+
+  describe('momentLangMap', () => {
+    it('should not map locales provided by upstream', () => {
+      const momentLangMap = config.get('momentLangMap');
+
+      Object.keys(momentLangMap).forEach((locale) => {
+        // If it does not throw, it means the file likely exists.
+        expect(() => {
+          // eslint-disable-next-line import/no-dynamic-require
+          require(`moment/locale/${locale}.js`);
+        }).toThrow();
```

Comment:
I am just not sure I understand this, should this throw on an invalid file? When I changed this to be a random file (that doesn't exist), it didn't throw an error so just double checking this..

Reply:
It is expected to throw, otherwise it means the locale file exists.

For example, if I change the test case like this:

```diff
diff --git a/tests/unit/test_config.js b/tests/unit/test_config.js
index 4a7030bc2..366b960ac 100644
--- a/tests/unit/test_config.js
+++ b/tests/unit/test_config.js
@@ -33,7 +33,8 @@ describe(__filename, () => {
     it('should not map locales provided by upstream', () => {
       const momentLangMap = config.get('momentLangMap');
 
-      Object.keys(momentLangMap).forEach((locale) => {
+      //Object.keys(momentLangMap).forEach((locale) => {
+      ['fr'].forEach((locale) => {
         // If it does not throw, it means the file likely exists.
         expect(() => {
           // eslint-disable-next-line import/no-dynamic-require

```

I get this error:

```
 FAIL  tests/unit/test_config.js
  ● /Users/williamdurand/projects/mozilla/addons-frontend/tests/unit/test_config.js › momentLangMap › should not map locales provided by upstream

    expect(function).toThrow(undefined)

    Expected the function to throw an error.
    But it didn't throw anything.

      40 |           // eslint-disable-next-line import/no-dynamic-require
      41 |           require(`moment/locale/${locale}.js`);
    > 42 |         }).toThrow();
         |            ^
      43 |       });
      44 |     });
      45 |   });

      at tests/unit/test_config.js:42:12
          at Array.forEach (<anonymous>)
      at Object.<anonymous> (tests/unit/test_config.js:37:14)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 2 passed, 3 total
```